### PR TITLE
Simulation

### PR DESCRIPTION
**Pull request by SimonR99**
- add simulation in dev container (windows support)
- add simulation file for rove (urdf)
- add default world

### CODE REVIEW

File: src/rove_description/package.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rove_description</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="capra@etsmtl.ca">user</maintainer>
```
Comment (Leuchak) [inline src/rove_description/package.xml:7]: capra@ens.etsmtl.ca

**Review by Leuchak — CHANGES_REQUESTED** [review]
How do I test this ?

**Comment by SimonR99** [comment]
I'll add a readme before merging. To this it meanwhile :

colcon build --symlink-install
source install/setup.bash
ros2 launch rove_description sim.launch.py

**Review by Leuchak — APPROVED** [review]
let's go

**Pull request merged**